dockerfiles/merge/merge.sh

```diff
@@ -2,30 +2,39 @@
 
 printUsage() {
    echo "Usage : "
-   echo "./merge.sh SHA1 RESULTING_BRANCH"
-   echo "    SHA1: SHA1 to merge with master"
+   echo "./merge.sh SHA1 RESULTING_BRANCH ORIGINAL_BRANCH"
+   echo "    SHA1: SHA1 to merge with the branch"
    echo "    RESULTING_BRANCH : Resulting branch of the merge"
+   echo "    ORIGINAL_BRANCH: the original branch used for merge (if none, then `master` will be used)"
    exit 1
 }
 
-if [ "$#" -ne 2 ]; then
+if [ "$#" -lt 2 ] || [ "$#" -gt 3 ]; then
     printUsage
 fi
 
 SHA1=$1
 RESULTING_BRANCH=$2
+if [ "$#" -eq 3 ]; then
+    ORIGINAL_BRANCH=$3
+else
+    ORIGINAL_BRANCH=master
+fi
 
 APACHE_REPO=`git remote show | grep apache || true`
 if [ -z "$APACHE_REPO" ]; then
     git remote add apache https://github.com/apache/james-project.git
 fi 
+# Getting original branch from apache repo
 git fetch apache
-git checkout apache/master -b master
+git checkout apache/$ORIGINAL_BRANCH -B $ORIGINAL_BRANCH
```

Comment:
I would have put the repo as part of the origin branch.
It would allow to have branches specific to Linagora.

Reply:
That's another story ;)

Reply:
why? what does it change to do it know?

I find it a shame that we need to change the api just after...